static/src/javascripts/projects/commercial/modules/header-bidding/prebid/appnexus.js

```diff
@@ -128,6 +128,7 @@ export const getAppNexusDirectBidParams = (
     };
 };
 
+// TODO are we using getAppNexusServerSideBidParams anywhere?
```

Comment:
That's for a different PR!

Reply:
Yeah I also created a trello card to check it: https://trello.com/c/AGIiBb1w/1057-check-unused-getappnexusserversidebidparams